community-cookbooks/memcached/recipes/configure.rb

```diff
@@ -19,9 +19,15 @@
 
 directory node['memcached']['logfilepath']
 
-service 'memcached' do
-  action :enable
-  supports :status => true, :start => true, :stop => true, :restart => true, :enable => true
+if node['memcached']['service_enabled']
```

Comment:
This condition should be checked inside each resource via `not_if` / `only_if` blocks to ensure the value is being read at the right time.